docs/topics/about-ide-addons.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+
+[id="about-ide-addons_{context}"]
+= About the {ProductShortName} add-ons for IDEs
+
+The {ProductShortName} add-ons for IDEs enable you to migrate and modernize applications while you work in a code editor.
+
+The add-ons analyze your projects using customizable rulesets, mark issues in the source code, provide guidance to fix the issues, and offer Quick Fixes, automatic code replacement, if possible.
+
+{ProductShortName} add-ons are available for the following IDEs:
+
+* Eclipse/Red Hat CodeReady Studio
```

Comment:
I know that Eclipse and RH CRS are basically the same but I think you should use "and" instead of "/". It just looks odd.